packages/instant/src/redux/async_data.ts

```diff
@@ -24,7 +24,9 @@ export const asyncData = {
             const errorMessage = 'Error fetching ETH/USD price';
             errorFlasher.flashNewErrorMessage(dispatch, errorMessage);
             dispatch(actions.updateEthUsdPrice(BIG_NUMBER_ZERO));
+            dispatch(actions.updateBaseCurrency(BaseCurrency.ETH));
```

Comment:
nice. this could be weird if we implement the eth price heartbeat but we will deal with that then.

Reply:
agreed -- in the future we should have a different error handling mechanism for when the fetch is triggered by the heartbeat